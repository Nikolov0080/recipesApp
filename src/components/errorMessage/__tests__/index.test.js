import React from 'react'
import ReactDOM from 'react-dom';
import ErrMsg from '../index';
import {  cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

afterEach(cleanup);

it('renders ErrMsg OK', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ErrMsg />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('matches snapshot', () => {
    const three = renderer.create(<ErrMsg text="test" variant="auth"/>).toJSON();
    expect(three).toMatchSnapshot()
});

it('matches snapshot', () => {
    const three = renderer.create(<ErrMsg text="test" variant="info"/>).toJSON();
    expect(three).toMatchSnapshot()
});

it('matches snapshot', () => {
    const three = renderer.create(<ErrMsg  variant="info"/>).toJSON();
    expect(three).toMatchSnapshot()
});