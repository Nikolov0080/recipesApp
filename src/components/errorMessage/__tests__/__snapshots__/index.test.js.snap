// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    className="fade err alert alert-danger show"
    role="alert"
  >
    test
  </div>
</div>
`;

exports[`matches snapshot 2`] = `
<div>
  <div
    className="fade info alert alert-info show"
    role="alert"
  >
    test
  </div>
</div>
`;

exports[`matches snapshot 3`] = `
<div>
  <div
    className="fade ok alert alert-success show"
    role="alert"
  >
    <h3>
      All good
      <span>
         âœ“
      </span>
    </h3>
  </div>
</div>
`;
